packages/scene-composer/src/components/panels/ColorPicker/ColorPickerHelpers.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { FormField, Input, InputProps } from '@awsui/components-react';
+import { ColorRepresentation } from 'three';
+import tinycolor from 'tinycolor2';
+import './ColorPicker.scss';
+import { HexColorPicker } from 'react-colorful';
+
+export const DEFAULT_COLOR = new tinycolor().toHexString();
+
+const toTinyColor = (color: ColorRepresentation) => {
+  return typeof color === 'number' ? new tinycolor(color.toString(16)) : new tinycolor(color);
+};
+
+export const hexString = (color: ColorRepresentation) => {
+  return toTinyColor(color).toHexString();
+};
+
+export const updateColorWithRgb = (oldColor: ColorRepresentation, newValue) => {
+  return new tinycolor({ ...toTinyColor(oldColor).toRgb(), ...newValue }).toHexString();
+};
+
+export const isValidColor = (color: ColorRepresentation) => {
+  return toTinyColor(color).isValid() && toTinyColor(color).getFormat() === 'hex';
+};
+
+export const validateHex = (value: string) => {
+  return `#${value.replace(/[^a-fA-F0-9]/g, '').substring(0, 6)}`;
+};
+
+export const getRgb = (color: ColorRepresentation) => {
+  return toTinyColor(color).toRgb();
+};
+
+export const validateRgbValue = (value: string) => {
+  const onlyDigits = value.replace(/\D/g, '');
+  return parseInt(onlyDigits) > 255 ? '255' : parseInt(onlyDigits.padStart(1, '0')).toString();
```

Comment:
Curious about 255 as a set cap here. Might be worth assigning to a `const` with a more meaningful name.

Reply:
Fixed